.circleci/config.yml

```diff
@@ -679,6 +817,14 @@ workflows:
             tags:
               only: /^v[0-9]+\.[0-9]+\.[0-9]+$/
 
+      # Publish Ralph helm chart to DockerHub
+      - publish-helm:
+          filters:
+            branches:
+              ignore: /.*/
+            tags:
+              only: /^helm-v.*/
```

Comment:
Why not a suffix instead (similar to warren tag patterns)?

Reply:
Wouldn't it be confusing to have helm chart tags and potential future plugin tags to be on the same level?
`v0.1.0-helm`
`v0.1.0-backend`
...

Reply:
Switched to a `vX.Y.Z-helm` pattern

Reply:
Although it conflicts with pre-release identifiers (`vX.Y.Z-beta1`), and they won't be taken into account anymore